registrations/tests/test_registration_get.py

```diff
@@ -221,6 +223,16 @@ def test_anonymous_user_can_see_registration(api_client, registration):
     assert_registration_fields_exist(response.data, is_admin_user=False)
 
 
+@pytest.mark.django_db
+def test_get_registration_contains_correct_signup_url(user_api_client, registration):
+    response = get_detail_and_assert_registration(user_api_client, registration.id)
+    for lang in ["en", "fi", "sv"]:
```

Comment:
Testing for complete equality (`response.json() == <urls_dict>`) would ensure that there are no extra keys/data

Reply:
Modified the test to assert that `signup_url` doesn't contain any extra fields